tests/test_import_data.py

```diff
@@ -298,12 +308,20 @@ def test_fix_attribution(connection_mock):
     assert result_json['award'] == 'test_award'
 
 
-# these tests will be replaced with dryrun tests
-
 @pytest.mark.file_operation
-def test_excel_reader_no_update_no_patchall_new_doc_with_attachment(capsys, mocker, connection_mock):
+def test_digest_xlsx(workbooks):
+    WORKBOOK_DIR = './tests/data_files/workbooks/'
+    for fn, workbook in workbooks.items():
+        book, sheets = imp.digest_xlsx(WORKBOOK_DIR + fn)
+        assert sheets == workbook.sheetnames
+        for sheet in sheets:
+            assert book[sheet].max_row == workbook[sheet].max_row
+            assert book[sheet].max_column == workbook[sheet].max_column
+
+
```

Comment:
Here and in the following tests the @pytest.mark was removed. It doesn't seem necessary, I just want to make sure it's intentional.

Reply:
it was as a fixture is used and no longer requires opening a file directly in the test.